sdk/containerregistry/azure-containerregistry/README.md

```diff
@@ -1,35 +1,105 @@
 
-This template package matches necessary patterns that the development team has established to create a unified sdk functional from Python 2.7 onwards. The packages contained herein can be installed singly or as part of the `azure` namespace. Any other introductory text should go here.
+Use the client library for Azure Container Registry to:
 
-This package has been tested with Python 2.7, 3.5, 3.6, 3.7 and 3.8
+- List images or artifacts in a registry
+- Obtain metadata for images and artifacts, repositories and tags
+- Set read/write/delete properties on registry items
+- Delete images and artifacts, repositories and tags
 
-For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
+[Source code][source] | [Package (Pypi)]<!--[package]--> | [API reference documentation]<!--[docs]--> | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
-# Getting started
+## Getting started
 
-For a rich example of a well formatted readme, please check [here.](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md) In addition, this is an [example readme](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-EXAMPLE.md) that should be emulated. Note that the top-level sections in this template align with that of the [template.](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md)
+### Install the package
 
-# Key concepts
+Install the Azure Container Registry client library for Python with [pip][pip_link]:
 
-Bullet point list of your library's main concepts.
+```bash
+pip install --pre azure-containerregistry
+```
 
-# Examples
+### Prerequisites
 
-Examples of some of the key concepts for your library.
+You need an [Azure subscription][azure_sub] and a [Container Registry account][container_registry_docs] to use this package.
 
-# Troubleshooting
+To create a new Container Registry, you can use the [Azure Portal][container_registry_create_portal],
+[Azure PowerShell][container_registry_create_ps], or the [Azure CLI][container_registry_create_cli].
+Here's an example using the Azure CLI:
 
-Running into issues? This section should contain details as to what to do there.
+```Powershell
+az acr create --name MyContainerRegistry --resource-group MyResourceGroup --location westus --sku Basic
+```
 
-# Next steps
+### Authenticate the client
 
-More sample code should go here, along with links out to the appropriate example tests.
+The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.
 
-# Contributing
+```python
+# Create a ContainerRegistryClient that will authenticate through Active Directory
+from azure.containerregistry import ContainerRegistryClient
+account_url = "https://MYCONTAINERREGISTRY.azurecr.io"
+client = ContainerRegistryClient(account_url, DefaultAzureCredential)
```

Comment:
It might be worth showing the `DefaultAzureCredential` import for clarity:
```suggestion
from azure.containerregistry import ContainerRegistryClient
from azure.identity import DefaultAzureCredential

account_url = "https://MYCONTAINERREGISTRY.azurecr.io"
client = ContainerRegistryClient(account_url, DefaultAzureCredential())
```